ui/pages/access-control/index.js

```diff
@@ -700,30 +700,59 @@ export default function AccessControl() {
           {
             id: 'delete',
             cell: function Cell(info) {
+              const [open, setOpen] = useState(false)
+              const { id, identityId, resource, privilege } = info.row.original
+
+              const name =
+                users?.find(u => u.id === identityId)?.name ||
+                groups?.find(g => g.id === identityId)?.name
+
               return (
                 isAdmin && (
```

Comment:
I'm not sure this is a good idea. It's confusing when an option is inexplicably disabled, doubly so when the option isn't even there

Reply:
this is actually a good point - i think instead of hiding it, its better to make it disabled

Reply:
hmm I am not sure what kind of confusion that you are referring to. The delete button will only be shown when the user is an admin. If the user is not an admin, she will not see the delete button nor be able to delete the grant. I think this is aligning with today's dashboard and on cli